lib/classes/plugin-manager.js

```diff
@@ -519,7 +522,16 @@ class PluginManager {
   }
 
   async runHooks(hookName, hooks) {
-    for (const { hook } of hooks) await hook();
+    const debugLog = log.get('lifecycle:command:invoke:hook');
```

Comment:
Was there a specific use case for introduction of this debug logic?

Reply:
Yes, in v2 I was relying on [debug](https://github.com/serverless/serverless/tree/debug) branch occasionally to see what lifecycle events were triggered.

We had requested to expose that also from the community: https://github.com/serverless/serverless/pull/7658

New logging systems enables to add that naturally to the core, and as I needed that when working on the console I've included this in this PR

Reply:
thanks for clarification, makes sense 👍